src/core/metrics/sources/nginx_access_log.go

```diff
@@ -350,6 +351,13 @@ func (c *NginxAccessLog) logStats(ctx context.Context, logFile, logFormat string
 }
 
 func getParsedRequest(request string) (method string, uri string, protocol string) {
+
+	re := regexp.MustCompile(`[0-9A-Za-z]+\s.+\s[0-9A-Za-z]+/[0-9A-Za-z]+`)
```

Comment:
When creating global variables with regular expressions you can use the MustCompile variation of Compile. MustCompile panics instead of returning an error, which makes it safer to use for global variables. - maybe use Compile instead?